odoo/modules/module.py

```diff
@@ -32,6 +32,7 @@
 
 # addons path as a list
 ad_paths = []
+ad_finders = []
```

Comment:
Seems redundant (and risky), shouldn't ad_finders *replace* ad_paths, and ad_paths either removed or a proxy of sort?

Reply:
I can retrieve the path with `ad_finder.path` but there is actually quite some modules that are using the `ad_paths` variable. I can be aggressive and change every `for path in ad_paths` by `for path in (finder.path for finder in ad_finders)`

Reply:
I meant that ad_paths could be a pseudo-list proxying to ad_finder. Though see main comment after more thinking mayhaps we can just strip out the whole thing e.g. warn on non-odoo.addons imports in 13.0, remove in master and have odoo.addons act as a more "proper" namespace without import hooks.